.github/workflows/integration-and-unit-tests.yml

```diff
@@ -10,7 +10,9 @@ on:
       - '**.yml'
       - '**.sh'
   pull_request:
-    branches: master
+    branches:
+      - master
+      - dev-3.8
```

Comment:
I suggest not to include this in this branch.

Reply:
And do it in a direct commit to `dev-3.8`?

Reply:
Reverted in [f3a21bf](https://github.com/polylang/polylang/pull/1642/commits/f3a21bf787bbdfdaf301d1ee53e09f8ed37890e1).